releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml

```diff
@@ -37,7 +37,7 @@ volumes:
 
 services:
   volume:
-    image: nexus3.edgexfoundry.org:10004/docker-edgex-volume:1.1.0
+    image: nexus3.edgexfoundry.org:10004/docker-edgex-volume:master
```

Comment:
We can make this change for all the repos but that means this docker compose file will be broken while DevOps is still rolling out the Geneva transformation. So we can either break it now or back it out have it updated during the transformation?

Reply:
Discussed out-of-band; believe you intend to add master label to existing builds.